index.html

```diff
@@ -39,17 +39,142 @@ <h3 class="connect">LET'S CONNECT</h3>
         <ul>
           <li class="social"><a href="#"><img src="./images/Vector.svg" alt="">
             </a></li>
-          <li class="social"><a href="#"><img src="./images/Linkedin icon.svg" alt="">
+          <li class="social"><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin">
             </a></li>
           <li class="social"><a href="#"><img src="./images/Vector (1).svg" alt="">
             </a></li>
-          <li class="social"><a href="#"><img src="./images/Vector (2).svg" alt="">
+          <li class="social"><a href="#"><img src="./images/Vector (2).svg" alt="github">
             </a></li>
-          <li class="social"><a href="#"><img src="./images/Vector (3).svg" alt="">
+          <li class="social"><a href="#"><img src="./images/Vector (3).svg" alt="twitter">
             </a></li>
         </ul>
       </div>
     </section>
+    <section class="works">
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio.png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>

```

Comment:
- [ Required ] I think it would be better if your design for the project cards would mirror the Figma template almost :100: Currently, there are parts that differ significantly from the Figma design provided for this project. The following areas are not significant with the Figma design, please improve them.
   
    - The top margin between the `works` section and the `introduction` section on your design is _too small_ compared to the margin on the Figma design.
    
   - There is no right margin on the project cards on your design whereas, on the Figma design, there is a margin on the _right-hand_ side of the cards.
    
    - The font size of the project titles is  _bigger_ in your design compared to the size of the same on Figma
   
    - The JavaScript text in the technologies section of your design overlaps the background color of the parent element whereas on Figma the text is within the background color of the parent element

|Figma design | Your design |
|-------------------|-----------------|
| ![Screenshot from 2023-06-03 02-02-15](https://github.com/MargaretTechworld/Portfolio/assets/16392046/52febed5-4f64-42b4-b627-471ca61fabbd)|![Screenshot from 2023-06-03 02-04-19](https://github.com/MargaretTechworld/Portfolio/assets/16392046/ce974817-9268-4dd1-93d5-79c0b123ebbd)|